west.yml

```diff
@@ -265,7 +265,7 @@ manifest:
         - hal
     - name: hostap
       path: modules/lib/hostap
-      revision: 4e4f601dae390263a91744db174fda4640d652cf
+      revision: pull/67/head
```

Comment:
please update the SHA as the PR is now merged.

Reply:
Thanks @krish2718 , SHA updated.